CMakeLists.txt

```diff
@@ -9,7 +9,11 @@ project(concord-bft VERSION 0.1.0.0 LANGUAGES CXX)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
+if(DEFINED IS_LEAKCHECK)
```

Comment:
Nit: Can we change the name of this option to be more declarative? Maybe just `LEAKCHECK`.

We probably should also default it to `TRUE`.

Reply:
Yes, will change the name. To default it to TRUE, though will require to fix all the tests, some are currently crashing with sanitizers enabled. The three I mentioned that helped me find leaks I have fixed to Pass.